app/tests/mock_database.py

```diff
@@ -762,6 +762,33 @@ def create_multiple_test_records():
     db.session.add(file_12)
     db.session.commit()
 
+    file_12_metadata = FileMetadata(
```

Comment:
What are these new objects for?

I'd suggest for new tests / changes to tests, we stop expanding and eventually stop using this function and use factory boy to create nested objects in addition to separating our test cases so that we dont need to have a huge test setup which is hard to debug

Reply:
I added this in as file_12 was missing valid metadata but yes we should switch to use factory boy for this.

Reply:
maybe a better question is which test(s) does this change impact?